lib/evaluateme.rb

```diff
@@ -1,4 +1,21 @@
+# Initializer file of the plugin.
+
 require "evaluateme/engine"
 
 module Evaluateme
+
+  # Secret key.
+  mattr_accessor :secret_key
+  @@secret_key = "evaluateme321"
+
+  mattr_accessor :parent_controller
+  @@parent_controller = "ApplicationController"
+
+  mattr_accessor :mailer_sender
+  @@mailer_sender = 'admin@evaluateme.com' 
```

Comment:
If [above comment][1] is correct, change  `@@mailer_sender` to `admin@example.com`


[1]https://github.com/RajRoR/evaluateme/pull/1/files#r28499132